### ISSUE TITLE
Playtest 20170930

### ISSUE THREAD

**Issue by dedmen** [blackhandcommando/openrabhc#9]
We are on playtest 20170408 as of #8.
Need to update...

**Comment by blackhandcommando** [comment]
Unfortunately neglected this project in favour of others. Just finished my server move and I think this weekend would be perfect for an upgrade.

**Comment by dedmen** [comment]
https://github.com/OpenRA/OpenRA/commits/bleed/mods/ra
From https://github.com/OpenRA/OpenRA/commit/6d84079a9480ae9e663865150b31ba9e5968073b#diff-112511e465a5d7ec30d26c3314657dcc to
To https://github.com/OpenRA/OpenRA/commit/eea14ec922ab091aaf829958333a9ee65179fe5d#diff-112511e465a5d7ec30d26c3314657dcc 37 sept

Those are a ton of commits to redo.. I'd say instead of applying their changes to our branch we just pull their current version and add our changes back onto it.
https://github.com/blackhandcommando/openrabhc/commits/next


https://github.com/blackhandcommando/openrabhc/commit/f96981aa8034e52d2770841b76960ad71ba42acd
https://github.com/blackhandcommando/openrabhc/commit/dfcc28cace860b23737d4cb8363942202450f7c5
#7
#5
#4
#6
https://github.com/blackhandcommando/openrabhc/commit/d96abc0d9f4b567e7c7680b5d965ded9ed16a5a5
https://github.com/blackhandcommando/openrabhc/commit/e5dac6e8323de190c2d25b2eb114b8f9f9b97412
https://github.com/blackhandcommando/openrabhc/commit/3f8f69e23ab62c6dd0214d861523f93d5c02f8e8

I'll start on it right away

**Comment by dedmen** [comment]
I just saw the stream.. the motd line-break problems... Use notepad++ and display line breaks.. You probably have  \r\n instead of \n.

**Comment by blackhandcommando** [comment]
That fixed it, thanks

**Comment by dedmen** [comment]
And here are the first problems
https://github.com/blackhandcommando/openrabhc/commit/e5dac6e8323de190c2d25b2eb114b8f9f9b97412#diff-24110d7b70209ba8d5d3f82f2c2fb6be
Changes SCUD range from 13 to 12. But in new playtest range is at 10.
It also decreases MIG view range from 15 to 14. But in new playtest it is already at 13.

**Comment by blackhandcommando** [comment]
Btw you are using 20170923 in the title reference, 20170930 is the latest

**Comment by dedmen** [comment]
Thanks. Should've checked.

**Comment by dedmen** [comment]
I think #10 is done now.

**Comment by dedmen** [comment]
They removed the modchooser. You now have to start the game with `OpenRA.exe Game.Mod=rabhc` apparently

**Comment by blackhandcommando** [comment]
Can you successfully host a server? Right now im getting https://imgur.com/a/y6EEp

**Comment by dedmen** [comment]
Jup.. I don't know where these differences came from. I got the ra folder from their github as basis. But there seem to be a lot of small differences between the github one and the one delivered in the playtest build. All fixed now

**Comment by blackhandcommando** [comment]
Getting this on my computer and server trying to start `OpenRA.exe Game.Mod=rabhc`:
```
OpenRA engine version playtest-20170930
Date: 2017-10-03 14:23:55Z
Operating System: Windows (Microsoft Windows NT 6.2.9200.0)
Runtime Version: .NET CLR 4.0.30319.42000
Exception of type `System.IO.FileNotFoundException`: File not found: common|chrome/ingame-player.yaml
   bei OpenRA.FileSystem.FileSystem.Open(String filename)
   bei OpenRA.WidgetLoader.<WidgetLoader>c__AnonStorey0.<>m__0(String a)
   bei System.Linq.Enumerable.WhereSelectArrayIterator`2.MoveNext()
   bei OpenRA.WidgetLoader..ctor(ModData modData)
   bei OpenRA.ModData..ctor(Manifest mod, InstalledMods mods, Boolean useLoadScreen)
   bei OpenRA.Game.InitializeMod(String mod, Arguments args)
   bei OpenRA.Game.Initialize(Arguments args)
   bei OpenRA.Program.Run(String[] args)
   bei OpenRA.Program.Main(String[] args)
```

Not sure why though


Edit: All done, sure its not working when not pulling the merge...

**Comment by dedmen** [comment]
That's correct. Thatf ile doesn't exist.. 
inagme-player is
rabhc|chrome/ingame-player.yaml
not
common|chrome/ingame-player.yaml

That's an error in mod.yaml. But.. I don't know why you have that. rabhc from my branch says
`rabhc|chrome/ingame-player.yaml`

OHHHH! You are testing your branch which is not updated.. instead of mine.


https://github.com/dedmen/openrabhc/archive/next.zip


Edit: :D

**Comment by dedmen** [comment]
Fixed by #10

**Comment by GraionDilach** [comment]
> I got the ra folder from their github as basis. But there seem to be a lot of small differences between the github one and the one delivered in the playtest build.

Sounds like someone downloaded the bleed branch instead of the prep-1707 branch. When the first playtest arrives, development branches off to a new prep branch and this branch will be the one which will end with the release.
